src/economy/flag.cc

```diff
@@ -361,7 +359,7 @@ Waterway* Flag::get_waterway(uint8_t const dir) const {
 /// \return the number of RoadBases connected to the flag
 uint8_t Flag::nr_of_roadbases() const {
 	uint8_t counter = 0;
-	for (uint8_t road_id = 6; road_id; --road_id) {
+	for (uint8_t road_id = WalkingDir::LAST_DIRECTION; road_id != WalkingDir::IDLE; --road_id) {
```

Comment:
I think these would look better like this:

```suggestion
	for (uint8_t road_id = WalkingDir::LAST_DIRECTION; road_id >= WalkingDir::FIRST_DIRECTION; --road_id) {
```

Reply:
Agreed and fixed.

Reply:
Thanks!